soh/soh/UIWidgets.hpp

```diff
@@ -36,7 +36,7 @@ namespace UIWidgets {
     bool EnhancementCheckbox(const char* text, const char* cvarName, bool disabled = false, const char* disabledTooltipText = "", CheckboxGraphics disabledGraphic = CheckboxGraphics::Cross);
     bool PaddedEnhancementCheckbox(const char* text, const char* cvarName, bool padTop = true, bool padBottom = true, bool disabled = false, const char* disabledTooltipText = "", CheckboxGraphics disabledGraphic = CheckboxGraphics::Cross);
     void EnhancementCombo(const std::string& name, const char* cvarName, const std::vector<std::string>& items, int defaultValue = 0);
-    void EnhancementCombobox(const char* name, const char* ComboArray[], size_t arraySize, uint8_t FirstTimeValue);
+    bool EnhancementCombobox(const char* name, const char* ComboArray[], size_t arraySize, uint8_t FirstTimeValue);
```

Comment:
i feel like having this be a bool is kind of odd, might make more sense to have it return an int because that's what the value in it is

Reply:
@briaguya-ai see the top resolved comment above, all of our wrapper ImGui components should be returning bools, consistent with ImGui and how it's supposed to be used